### PR TITLE
New package: waitforfile-0.1.7

### PR DESCRIPTION
**Pull request by cr6git**
[waitforfile](https://github.com/nroi/waitforfile)
Very useful !

### CODE REVIEW

**Comment by Vaelatern** [comment]
Well that is a new approach to cross compiling rust.

**Comment by pullmoll** [comment]
Hmm.... **Bahnhof** @Vaelatern ***nod and agree*** Is this for real? Reinterpreted:
```
if [ -z "$CROSS_BUILD" ]; then ...
    # Compile from source for all other archs
    ...
fi
```

I guess I should have been drinking a lot less than I did. Or is it that just my alternate reality bit flipped again? *lol* Perhaps I should get sober and take another look at that template again :)

**Comment by Vaelatern** [comment]
I don't know: is it better to have coverage at the expense of actually building the binaries ourselves?

**Comment by pullmoll** [comment]
I don't know either. It's just that the template looks very confusing to me.
And as per coverage I prefer binaries built by us, but in case that's not possible I think it is ok to
use binary distributions in cases like this.

File: srcpkgs/waitforfile/template [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+do_build() {
+	if [ -z "$CROSS_BUILD" ]; then
+		cargo build --release
+	else
+		printf "\033[1;32m=> precompiled binaries, nothing to do.\033[m\n"
+	fi
+}
+
+do_install() {
+	if [ -z "$CROSS_BUILD" ]; then
```
Comment (Vaelatern) [inline srcpkgs/waitforfile/template]: Maybe you can move this logic out of the functions? To try and simplify the template.

**Comment by cr6git** [comment]
@pullmoll Changes were made to make it more clear!

**Comment by pullmoll** [comment]
This is actually close to what I can grok. I @Vaelatern is also happy he could merge?

**Pull request merged**